### PR TITLE
Admin enables merchant

### LINKED ISSUE #40: User Story 40, Admin enables a merchant account
As an admin
When I visit the merchant index page
I see an "enable" button next to any merchants whose accounts are disabled
When I click on the "enable" button
I am returned to the admin's merchant index page where I see that the merchant's account is now enabled
And I see a flash message that the merchant's account is now enabled

### PR DESCRIPTION
**Pull request by KieraAllen**
This PR:

- does basically the same things as PR #79 but in reverse!
- closes #40

### CODE REVIEW

**Review by JoseLopez235 — APPROVED** [review]

File: app/controllers/admin/merchants_controller.rb [hunk 1]
```diff
@@ -11,4 +11,11 @@ def disable
     redirect_to "/admin/merchants"
     flash[:notice] = "#{@merchant.name} is disabled"
   end
+
+  def enable
+    @merchant = Merchant.find(params[:merchant_id])
```
Comment (stillsheryl) [inline app/controllers/admin/merchants_controller.rb:16]: Does this need to be an instance variable?

Comment (KieraAllen) [reply app/controllers/admin/merchants_controller.rb:16]: Great question... let me check.

Comment (KieraAllen) [reply app/controllers/admin/merchants_controller.rb:16]: It doesn't! I'll change that when I do US #41

File: spec/features/admin/merchants/index_spec.rb [hunk 2]
```diff
@@ -93,5 +93,64 @@
...
+
+    it "I see an 'Enable' button next to any disabled merchants" do
+
+      @bike_shop.update_attribute(:active?, false)
+      @fishing_shop.update_attribute(:active?, false)
+
+      visit "/admin/merchants"
+
+        within "#merchant-#{@bike_shop.id}" do
+          expect(page).to_not have_button("Disable")
```
Comment (c-bartell) [inline spec/features/admin/merchants/index_spec.rb:105]: Great job writing thorough tests!

**Review by c-bartell — APPROVED** [review]
Looks good!

**Pull request merged**